dev/large-requirements.txt

```diff
@@ -2,3 +2,4 @@
 matplotlib
 plotly
 tf2onnx
+azure-storage-blob>=12.0.0
```

Comment:
@WeichenXu123 To clarify, we've added this as a test requirement because it's required for `azure_blob_artifact_repo.py` right?